.vscode/settings.example.json

```diff
@@ -1,4 +1,5 @@
 {
+  "eslint.experimental.useFlatConfig": true,
```

Comment:
Heads up for contributors:
If you are using VSCode, please add this line to your project config after this PR gets merged, otherwise the linter extension will not work.